src/index.ts

```diff
@@ -15,12 +27,15 @@ export default class TabContainerElement extends HTMLElement {
       if (target.getAttribute('role') !== 'tab' && !target.closest('[role="tablist"]')) return
       const tabs = getTabs(this)
       const currentIndex = tabs.indexOf(tabs.find(tab => tab.matches('[aria-selected="true"]'))!)
+      const [incrementKey, decrementKey] = getNavigationKeys(
+        !!target.closest('[role="tablist"][aria-orientation="vertical"]')
```

Comment:
Would this return `true` if there was a non-vertical tablist nested inside a vertical tablist? And wouldn't you want it to return `false` in that case?

What do you think of something like this?
```suggestion
        !!(target.closest('[role="tablist"]')?.getAttribute('aria-orientation') === 'vertical')
```

If that seems correct, would you mind adding a test for that case? It not, I'm happy to chat about other options (or figure out my misunderstanding)

Reply:
@jfuchs thank you for the review! I understand the implementation change, but I'm unsure about how I'd write a test to nest in the way you're describing. Would you be interested in sharing pseudo code for the example?